src/components/common/WorkCard/styles.js

```diff
@@ -0,0 +1,16 @@
+import styled from 'styled-components'
+
+import { colors } from '../../../utils'
+
+export const WorkWrapper = styled.div`
+  padding: 3.5rem;
+  background-color: ${colors.primary};
+  width: 100%;
+  height: 710;
```

Comment:
this doesnt work bc it doesnt have a unit 
![image](https://user-images.githubusercontent.com/8256872/76644163-22e5b780-655f-11ea-9ef9-91607640e7ad.png) 
and either way i dont think using a fixed height is a good thing